dora/core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -1310,6 +1393,12 @@ public Command workerHeartbeat(long workerId, Map<String, Long> capacityBytesOnT
     // by the LostWorkerDetectionHeartbeatExecutor
     worker.updateLastUpdatedTimeMs();
 
+    if (mWorkerRegisterToAllMasters && mPrimarySelector.getState() == NodeState.STANDBY) {
```

Comment:
let's comment out this one? this is not needed in dora as blocks are not written as journals. Waiting on this might cause some unexpected behavior